image-layout.md

```diff
@@ -15,9 +15,12 @@ The image layout has two top level directories:
 - "refs" contains [descriptors][descriptors]. Commonly pointing to an [image manifest](manifest.md#image-manifest) or an [image manifest list](manifest-list.md#oci-image-manifest-list-specification).
 
 
-It also contains a file that is used to identify the layout version:
+The image layout MUST contain an `oci-layout` file:
 
-- "oci-layout" MUST contain a JSON object with a version field `{"imageLayoutVersion": "1.0.0"}` and MAY include additional fields.
+- It MUST be a JSON object
+- It MUST contain an `imageLayoutVersion` field
+    - The `imageLayoutVersion` value will align with the OCI Image Specification version at the time changes to the layout are made, and will pin a given version until changes to the layout are required
```

Comment:
This file is about the OCI Image Layout Specification, and if `imageLayoutVersion` semantically versions that, I'm happy.  However, it seems like the “pin” language was adding a relationship to the project-wide OCI Image Format Specification version.

To distinguish the two cases, pretend we had already cut 1.0.0, and say there is a backwards-compatible change to the image layout (e.g. you add an optional index, #438) and a backwards-incompatible change to something else (e.g. we drop support for `.wh.*`, #24).  The backwards-incompatible change means the project as a whole goes to 2.0.0.  With the independently-SemVer'ed approach, the backwards-compatible image-layout change moves `imageLayoutVersion` to 1.1.0.  With the pinned-to-the-project-version approach, `imageLayoutVersion` moves to 2.0.0.  Which of those is your intended result?